test/functional/mempool_compatibility.py

```diff
@@ -33,18 +33,24 @@ def setup_network(self):
             None,
         ])
         self.start_nodes()
-        self.import_deterministic_coinbase_privkeys()
 
     def run_test(self):
         self.log.info("Test that mempool.dat is compatible between versions")
 
-        old_node = self.nodes[0]
-        new_node = self.nodes[1]
-        recipient = old_node.getnewaddress()
-        self.stop_node(1)
+        old_node, new_node = self.nodes
+        old_wallet = MiniWallet(old_node)
+        new_wallet = MiniWallet(new_node)
+
+        # Add enough mature utxos to the wallets so that all txs spend confirmed coins.
```

Comment:
No need for this comment? There is only one inputs, so this is clear from reading the code.